CHANGELOG.md

```diff
@@ -1,3 +1,18 @@
+# EC2 Rescue for Linux v1.1.6
```

Comment:
The current development version is 1.1.5. Please add your change notes to 1.1.5.